WordPress/src/main/res/values/strings.xml

```diff
@@ -328,6 +328,9 @@
     <string name="pages_fetching">Fetching pages…</string>
     <string name="toast_err_post_uploading">Unable to open post while it\'s uploading</string>
     <string name="dialog_confirm_cancel_post_media_uploading">Are you sure? Deleting this post will also cancel the media upload.</string>
+    <string name="dialog_confirm_publish_title">Ready to Publish?</string>
+    <string name="dialog_confirm_publish_message">This %s will be published immediately.</string>
```

Comment:
Please create two separate strings:
`This post will be published immediately` and `This page will be published immediately`. "This" might need different translations, depending on the gender of page or post.

Reply:
oh that's right. Awesome, thank you! Addressed in c8803fc 🙇